targets/zephyr/Makefile.zephyr

```diff
@@ -70,7 +70,7 @@ CONFIG_TOOLCHAIN_VARIANT = x86
 CPU = x86
 EXT_CFLAGS += -march=pentium
 EXT_CFLAGS += -mpreferred-stack-boundary=2 -mno-sse
-else ifeq ($(BOARD),qemu_cortex_m3)
+else ifeq ($(BOARD),$(filter $(BOARD),qemu_cortex_m3 frdm_k64f))
 CONFIG_TOOLCHAIN_VARIANT = arm
```

Comment:
Might be worth to add a separate case for the FRDM-K64F board with more aggressive compiler flags, tuning for the Cortex-M4F to allow the compiler to take advantage of the native DSP and floating-point instructions which are available with ARMv7E-M, e.g. -mcpu=cortex-m4 -mfpu=fpv4-sp-d16.  
Native floating-point support will definitely make a performance difference for JerryScript :)